modules/storage.sh

```diff
@@ -0,0 +1,151 @@
+function storage_total() {
+  option=$1
+  case $option in
+    'gb')
+      t_M=$(df -BG | grep '/dev/root' | awk '{print $2}')
+      t=$(echo "scale=2;$t_M/(1024*(10^-2))" | bc)
```

Comment:
I think `df -BG | grep '/dev/root' | awk '{print $2}'` already gives the result, why need to recalculate here and below.

Reply:
Without the recalculation, the output in gigabytes in incorrect